src/user.js

```diff
@@ -21,22 +23,42 @@ class User {
 	}
 	
   searchForRecipe(keyword, array) {
+    let byName = this.searchByName(keyword, array)
+    let byIngredient = this.searchByIngred(keyword, array)
+    let searchedResults = byName.concat(byIngredient)
+    console.log('results', searchedResults)
+    return [... new Set(searchedResults)]
+  }
```

Comment:
this block, const instead of let again. also, if our team wants to keep with semi-colons after each line, would be good for consistency to fix that here